source/server/listener_impl.h

```diff
@@ -224,13 +225,15 @@ class ListenerImpl : public Network::ListenerConfig,
   const uint64_t hash_;
   ProtobufMessage::ValidationVisitor& validation_visitor_;
 
+  // This init watcher, if available, notifies the "parent" listener manager when listener
+  // initialization is complete. It may be reset to cancel interest.
+  std::unique_ptr<Init::WatcherImpl> init_watcher_;
```

Comment:
I know it was already like this, but I don't think this needs to be a `unique_ptr`.

Reply:
done